cloudslang-cli/src/main/java/io/cloudslang/lang/cli/SlangCLI.java

```diff
@@ -103,10 +104,11 @@ public String setEnvVar(
 
     @CliCommand(value = "inputs", help = "Get flow inputs")
     public List<String> getFlowInputs(
-            @CliOption(key = {"", "f", "file"}, mandatory = true, help = "Path to filename. e.g. cslang inputs --f C:\\CloudSlang\\flow.yaml") final File file,
-            @CliOption(key = {"cp", "classpath"}, mandatory = false, help = "Classpath , a directory comma separated list to flow dependencies, by default it will take flow file dir") final List<String> classPath)
+            @CliOption(key = {"", "f", "file"}, mandatory = true, help = "path to filename. e.g. cslang inputs --f C:\\CloudSlang\\flow.yaml") final File file,
+            @CliOption(key = {"cp", "classpath"}, mandatory = false, help = "classpath , a directory comma separated list to flow dependencies, by default it will take flow file dir") final List<String> classPath,
```

Comment:
same, but the --addcp is used, it will add the directories to the default classpath instead of replacing it